### ISSUE TITLE
incorrect published sources

### ISSUE THREAD

**Issue by crafter312** [kyrptonaught/customportalapi#66]
I've been using customportalapi for a mod I'm working on using Intellij, but every time I try to follow the debugger into the source files for the api it complains that the "source code does not match the bytecode." Additionally, when I try to put a breakpoint inside the source files for the api, the debugger complains that there is no executable code at the breakpoint, even though the file viewer shows that there clearly is. I don't have this problem with any of my other dependencies. I talked with someone in the Modded Minecraft discord about this issue, and they informed me that it is likely that customportalapi does not have the correct sources published, or something like that.

It's always possible that this is just some weird issue with the way my project is set up, but I wanted to open an issue here in case it isn't. For reference, my mod can be found [here](https://gitfront.io/r/crafter312/6JEYGLepwrJA/fabric-backrooms-1.19.2/).

**Comment by kyrptonaught** [comment]
Hm, I'm really not sure what's causing this. I get this issue when viewing vanilla source too.

**Comment by crafter312** [comment]
You're right. I initially thought that the rest of my dependencies were fine and that it was just the custom portal api. However, I just went through and tested some of my other dependencies again, and I have the same issue now with my other dependencies as well. I ended up just making a copy of the class I wanted to debug, which I needed to modify anyways, so I guess this isn't actually an issue with the custom portal api.
